core/cloudflow-operator/src/test/scala/cloudflow/operator/runner/SparkRunnerSpec.scala

```diff
@@ -115,6 +115,40 @@ class SparkRunnerSpec extends WordSpecLike with OptionValues with MustMatchers w
       crd.spec.monitoring.prometheus.configFile mustBe PrometheusConfig.prometheusConfigPath(Runner.ConfigMapMountPath)
     }
 
+    "read labels for pod and adding them to driver and executor" in {
```

Comment:
Same here as comment in Flink but for driver, executor, driver and executor.

Reply:
Done